puppet/modules/slave/templates/pbuilder_F60addforemanrepo.erb

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Add the right theforeman.org repositories
+
+set -eu
+
+if [ -z "${FOREMAN_REPO_SETUP:-}" ]; then
```

Comment:
What's the case to skip this?

Reply:
https://github.com/theforeman/foreman-infra/pull/1643#discussion_r698433934

My idea was to easily switch between this and the "jenkins sets up things" for now, and remove later.